source/config/__init__.py

```diff
@@ -64,7 +64,13 @@ def isInstalledCopy():
 
 def getInstalledUserConfigPath():
 	try:
-		return os.path.join(shlobj.SHGetFolderPath(0, shlobj.CSIDL_APPDATA), "nvda")
+		k = _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE, NVDA_REGKEY)
+		configInLocalAppData = bool(_winreg.QueryValueEx(k, u"configInLocalAppData")[0])

```

Comment:
I honestly do not see why we should hard code here. In installer.py, NVDA_REGKEY isn't used either. Or do you want me to change all places where software\\nvda is used to config.NVDA_REGKEY?